sklearn/cross_decomposition/pls_.py

```diff
@@ -234,7 +234,7 @@ def fit(self, X, Y):
 
         Parameters
         ----------
-        X : array-like, shape = [n_samples, n_features]
+        X : array-like of predictors, shape = [n_samples, n_features]
```

Comment:
This line is intended for type specification only. "predictors" is semantics, not type. Also, we tend to use "features"/"target" rather than "predictors"/"response".